doc/frequency_evaluation_framework.md

```diff
@@ -0,0 +1,373 @@
+
+## Frequency Evaluation Framework
+Matthew Clegg, Google
+
+6/26/2020
+
+
+### Objective
+
+Propose changes to the WFA cardinality evaluation framework to enable evaluation of frequency estimators.
+
+
+### Background
+
+Google and Facebook are developing a cross media open measurement system in accordance with the World Federation of Advertisers.  As part of this effort, research is being conducted into methods for computing cross media reach and frequency in an efficient and differentially private manner.  In the [Cardinality and Frequency Estimation Evaluation Framework](https://github.com/world-federation-of-advertisers/cardinality_estimation_evaluation_framework/blob/master/doc/cardinality_and_frequency_estimation_evaluation_framework.md) document, several promising approaches were identified and criteria proposed for the evaluation and comparison of these approaches.  A [workbench](https://github.com/world-federation-of-advertisers/cardinality_estimation_evaluation_framework) was then developed for the evaluation of cardinality (reach) estimators.  The purpose of this doc is to propose how to extend this effort to include support for evaluation of frequency estimators.
+
+
+### Requirements
+
+
+
+*   Must support each of the scenarios for frequency evaluation envisioned in the [Cardinality and Frequency Estimation Evaluation Framework](https://github.com/world-federation-of-advertisers/cardinality_estimation_evaluation_framework/blob/master/doc/cardinality_and_frequency_estimation_evaluation_framework.md) document.
+*   Design should be as similar as possible to the current cardinality estimation framework.
+*   Output should be as similar as possible to the output of the current cardinality estimation framework.
+*   Should be designed in such a way that it can easily be parallelized to allow for running in Borg or Google Cloud.
+*   Should support frequency evaluators which are represented as a collection of cardinality sketches, as well as evaluators whose sketches can report frequency directly.
+
+
+### Out of Scope
+
+
+
+*   Implementation of the VoC and ADBF frequency estimation sketches.
+
+
+### System Diagram
+
+![System diagram of Evaluation Framework](img/evaluation-framework-diagram.png)
+
+A system diagram of the evaluation framework is given above. This figure applies
+both to the existing cardinality estimation framework as well as to the planned
+frequency estimation framework. The major components are as follows:
```

Comment:
This is great! We should consider add the diagram to the top level readme file as well.

Reply:
If it's OK with you, I will leave that for a subsequent pull request.

Reply:
This is fine. We can do this later.